src/main/flight/pid.h

```diff
@@ -238,7 +238,7 @@ typedef struct pidProfile_s {
     uint8_t angle_feedforward_smoothing_ms; // Smoothing factor for angle feedforward as time constant in milliseconds
     uint8_t angle_earth_ref;                // Control amount of "co-ordination" from yaw into roll while pitched forward in angle mode
     uint16_t horizon_delay_ms;              // delay when Horizon Strength increases, 50 = 500ms time constant
-    uint8_t tpa_low_rate;                   // Percent reduction in P or D at zero throttle
+    int8_t tpa_low_rate;                    // Percent reduction in P or D at zero throttle
```

Comment:
Requires incrementing PID PG version.

Reply:
tpa was limited to 0..100, int and uint is identical for this range